README.md

```diff
@@ -1,122 +1,91 @@
-We envision a vibrant, dynamic library of open-access ARCO data stored in public clouds, shared among thousands of scientists and directly accessible to data-proximate computing.
-However, manually populating such a library would be prohibitively difficult and tedious.
-Instead, we are building Pangeo Forge to automate the production of ARCO data and enable the croudsourcing of such a data library.
-
-In Pangeo Forge, a maintainer contributes a recipe which is used to generate an analysis-ready cloud-based copy of a dataset in a cloud-optimized format like Zarr. Behind the scenes, Pangeo Forge cloud-based automation downloads the original files from their source (e.g. FTP, HTTP, or OpenDAP), combines them into one coherent dataset (e.g. using xarray), and writes the data in a cloud optimized format (e.g. Zarr) to cloud storage in a streaming fashion.
-
-## Technical Concepts and Architecture
-
-:exclamation: **Warning!** Pangeo Forge doesn't actually "work" yet. The integration and development of these compoments is work in progress.
-
-### Recipes
-
-A recipe defines how to transform data in one format / location into another format / location.
-The primary way people contribute to Pangeo Forge is by writing / maintaining recipes.
-Recipes are python objects generated by the [pangeo_forge](https://pangeo-forge.readthedocs.io/en/latest/) package.
-These recipes can be used in a standalone fashion, without integration with the Pangeo Forge cloud automation infrastructure.
-Or they can be turned into feedstocks and become part of the library.
+Pangeo Forge aspires to be like Conda Forge, but for data — specifically, for Analysis Ready, Cloud Optimized (ARCO) data. For a detailed working definition of ARCO data, see our paper [Cloud Native Repositories for Big Scientific Data](https://ieeexplore.ieee.org/abstract/document/9354557).
 
-### Feedstocks
-
-Feedstocks are recipes that are managed and executed by Pangeo Forge cloud automation.
-Feedstocks are stored in GitHub repositories in the [pangeo-forge GitHub organization](https://github.com/pangeo-forge/).
-The community develops and maintains recipes through interaction with these repositories.
-
-### Bakeries
-
-Bakeries turn recipes into data.
-They do the heavy lifting of actually executing the recipes: extracting data from its source, transforming it, and loading it into its target destination.
-Bakeries are controlled by triggers from GitHub workflows.
-Bakeries can run in cloud or on-premises compute nodes; they should be placed in close network proximity to data sources and / or targets.
-We hope that eventually there will be Pangeo Forge bakeries running in most regions of major cloud providers.
-
-![diagram](pangeo-forge-diagram.png)
+We envision a vibrant, dynamic library of open-access ARCO data stored in public clouds, shared among thousands of scientists and directly accessible to data-proximate computing.
```

Comment:
I like keeping this as the vision, even if the current implementation is a bit more distributed in terms of ownership.